__tests__/endpoints.test.js

```diff
@@ -136,4 +136,47 @@ describe('Test Endpoints', () => {
             });
         }); 
     });
+
+    //POST
+    describe('/api/articles/:article_id/comments', () => {
+        test("201 - should add a comment to the relevant article_id", () => {
+            const newComment = {
+              body: "The Worlds First Photo of Quantum Entanglement Could Disprove Einsteins Theory",
+              article_id: 1,
+              author: "butter_bridge",
+              votes: 0,
+              created_at: new Date(),
+            };
+            return request(app)
+              .post("/api/articles/1/comments")
+              .send(newComment)
+              .expect(201)
+              .then(({ body }) => {
+                const comment = body.comment;
+                expect(typeof comment.comment_id).toBe("number");
+                expect(typeof comment.body).toBe("string");
+                expect(comment.article_id).toBe(1);
+                expect(typeof comment.author).toBe("string");
+                expect(comment.votes).toBe(0);
+                expect(typeof comment.created_at).toBe("string");
+              });
+          });
+      
+          test("POST: 400 - should return with appropriate error when comment is blank", () => {
+            const newComment = {
+              body: null,
+              article_id: 1,
+              author: "butter_bridge",
+              votes: 0,
+              created_at: new Date(),
+            };
+            return request(app)
+              .post("/api/articles/1/comments")
+              .send(newComment)
+              .expect(400)
+              .then((response) => {
+                expect(response.body.msg).toBe("Body cannot be blank");
+              });
+          });
+    }); 
```

Comment:
Need another test 400 missing required key - newComment = {}
Also need 404 article not found and 400 invalid article id